README.md

```diff
@@ -1,135 +1,277 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/98ac537e-14f6-4864-bf56-d5a60c76ccc9/deploy-status)](https://app.netlify.com/sites/ss-docs/deploys)
+# Silverstripe CMS Documentation
 
-# doc.silverstripe.org
+Next.js code for [doc.silverstripe.org](https://doc.silverstripe.org) and [userhelp.silverstripe.org](https://userhelp.silverstripe.org).
 
-This repository contains the source code powering [the Silverstripe CMS
-developer documentation website](https://docs.silverstripe.org) and
-[userhelp website](https://userhelp.silverstripe.org).
+Built with:
+- Next.js 16+
+- Node.js 24+
+- TypeScript
+- Markdown (remark/rehype)
+- Algolia DocSearch
+- Jest + RTL
 
-This application is build on [Gatsby](https://gatsbyjs.com), a static
-site generator based on [React](https://reactjs.org). It sources content
-from the [silverstripe/developer-docs](https://github.com/silverstripe/developer-docs)
-repository for each major release.
+---
 
-**This repository does NOT contain any documentation.**
+## Quick Start
 
-The developer documentation is stored in the [silverstripe/developer-docs repository](https://github.com/silverstripe/developer-docs)),
-in the `en` folder. For example, the documentation for the
-Silverstripe CMS 4.x is stored in
-[https://github.com/silverstripe/developer-docs/tree/4/en](https://github.com/silverstripe/developer-docs/tree/4/en).
+```bash
+npm ci                 # Install dependencies (use npm ci, not npm update)
 
-The userhelp documentation is stored in the [silverstripe/silverstripe-userhelp-content repo](https://github.com/silverstripe/silverstripe-userhelp-content/).
+# Development with mock data (recommended for development)
+npm run mock           # Dev server with mock data (no cloning needed)
 
-## What to update when creating a new pre-release major branch, making a stable major release, or making a major EOL
+# Development with real cloned content
+npm run dev:docs       # Dev server with developer docs on port 9876
+npm run dev:user       # Dev server with user help on port 9876
+npm run clone:docs     # Clone developer docs without starting dev server
+npm run clone:user     # Clone user help docs without starting dev server
 
-When **creating a new major branch for a pre-release major version**
+# Production build & test
+npm run build          # Build & lint (default: docs context, alias for build:docs)
```

Comment:
```suggestion
npm run build          # Build, lint, and test (default: docs context, alias for build:docs)
```
If it doesn't currently run the tests, it should be updated to run the tests, consistent with every other `build` js script we run.